tests/stub/datatypes/test_unsupported_type.py

```diff
@@ -53,110 +53,88 @@ def _session(self, driver):
         finally:
             session.close()
 
-    def test_unknown_type_subtests(self):
+    def test_unsupported_type_subtests(self):
```

Comment:
```suggestion
    def test_unsupported_type(self):
```
or what is the thinking behind the "subtests" suffix?

Reply:
A remnant from when I converted these tests from separate but almost identical to subtests, no reason to have it there anymore. thanks!